fetch.ts

```diff
@@ -85,7 +124,7 @@ export function isValidRedirection(from: URL, to: URL) {
       (from.host === to.host && from.pathname === to.pathname && from.searchParams.size !== to.searchParams.size) ||
       // Login redirections; e.g., Firebase Console -> Google Account Login
       (
-        (to.hostname === "accounts.google.com" && segments.to[2] === "signin") || // Google
+        (to.hostname === "accounts.google.com" && (segments.to[2] === "signin" || segments.to[1] === "signup")) || // Google
```

Comment:
We seems to get different paths from that redirect.

> The redirect in my browser is https://accounts.google.com/signup/v2/createaccount?theme=glif&flowName=GlifWebSignIn&flowEntry=SignUp.

> Mine actually is https://accounts.google.com/lifecycle/steps/signup/name with some long query parameters added to that.

Whatever the path is, it seems to have the "signin" or "signup" somewhere, not necessarily at the beginning. So, I think a simple `.includes` check would do.

```suggestion
        (to.hostname === "accounts.google.com" && (segments.to.includes("signin") || segments.to.includes("signup"))) || // Google
```